src/openloco/interop/hook.cpp

```diff
@@ -190,7 +191,7 @@ namespace openloco::interop
         register_hook(
             address,
             [](registers& regs) -> uint8_t {
-                std::printf("                    fn %u\n", passAddress);
+                std::printf("                    fn %08" PRIXPTR "\n", passAddress);
```

Comment:
I'd expect `PRIXPTR` to already contain zero-padding. Otherwise I'd suggest going with `%p`.

Reply:
Eh? `PRIXPTR` is merely the `X` or `lX` or `llX` or whatever suffix of the format specifier that's needed for `uintptr_t`, nothing more.

I'm actually a big anti-fan of `%p`, because (a) it's implementation-defined, and (b) I've seen it do stupid things like use different formatting for null addresses than for non-zero addresses for some stupid reason. Makes things end up being ugly.